src/client/src/css/jobListing.css

```diff
@@ -253,53 +190,173 @@
   font-size: large;
 }
 
-@media all and (max-width: 690px) {
-  .jobPosts_Container {
+@media all and (max-width: 905px) {
+  .jobPosts {
+    border: 2px #19718d solid;
+    width: 85%;
+    height: 570px;
+    border-radius: 15px;
+    margin: auto;
+    margin-top: 30px;
+    display: inline-block;
+  }
+
+  .jobPosts .AlertJobListing{
+    display: none;
+  }
+  .jobPosts .YetAppliedJobs{
+    display: none;
+  }
```

Comment:
There seems to be quite some wasted space when the screen is in between 686 and 905 pixels
![image](https://user-images.githubusercontent.com/78051064/230804156-79d8c595-d56f-4f4e-9d6e-5f84a15d0f01.png)